Kickstarter-iOS/Views/Controllers/MessageBannerViewController.swift

```diff
@@ -124,7 +124,9 @@ final class MessageBannerViewController: UIViewController, NibLoading {
         if isHidden {
           // Tells Voice Over to resign focus of the message banner.
           // This causes the reader to focus on the previously selected element.
-          UIAccessibility.post(notification: UIAccessibility.Notification.layoutChanged, argument: nil)
+          if AppEnvironment.current.isVoiceOverRunning() {
```

Comment:
I've placed these under `isVoiceOverRunning()` check in order to make this consistent thorough the code base